docs/source/developer.adoc

```diff
@@ -1,3 +1,60 @@
 = Developer Documentation
+:imagesdir: ./assets
+:toc:
 
-This is the developer documentation for fabric8-auth.
\ No newline at end of file
+This is the developer documentation for fabric8-auth. It is useful if you wish to contribute to the fabric8-auth project in some way.
+
+== Source code
+
+The latest source code for fabric8-auth can be found at GitHub link:https://github.com/fabric8-services/fabric8-auth[here].  The repository is configured to perform automatic, continual releases from the *master* branch any time a new pull request is merged.
+
+== Issue Tracking
+
+link:https://github.com/fabric8-services/fabric8-auth/issues[GitHub issues] are used for tracking the development of new features and service issues.
+
+== Development workflow
+
+Basically, all work done to the fabric8-auth code base must have a corresponding issue, whether it be a complex new feature, major or minor bug or simply correcting a typo in the code, readme file or documentation.  This allows the project team to carefully manage and audit the changes made to such a critical piece of the fabric8 platform.  If you wish to contribute any code or other changes to the project, here are the steps to follow:
+
+. Pick an existing issue to work with (or create a new one)
+. Fork the fabric8-auth repository to your own GitHub account
+. Check out the forked repository in your development environment
+. Create a new branch representing the issue you will work on (*git checkout -b ISSUE-XXX*)
+. Make your changes, commit them to the issue branch
+. Push the changes to your forked repository
+. In Github, browse to the issue branch in your forked repository
+. Create a pull request for your issue branch, making sure to clearly link the issue that the PR is for
+. Update the issue with a link to the pull request
+
+== Service Accounts
+
+Service accounts are used by resource servers to interact with the fabric8-auth protection API.  This API requires a PAT (Protection API Token) in order for the resource servers to register newly created resources, and query the access rights for clients attempting to access resources that they don't own.
+
+Service account configuration is expected to be found in */etc/fabric8/service-account-secrets.conf*, in JSON format.  Here's an example configuration:
+
+[source,json]
+{
+    "accounts": [
+        {
+            "name":"fabric8-wit",
+            "id":"5dec5fdb-09e3-4453-b73f-5c828832b28e",
+            "secrets":["witsecret"]
```

Comment:
Let's use hashes in the example. Plain text secrets won't work. We expect a bcrypt hash in the config file.